internal/rbd/nodeserver.go

```diff
@@ -701,7 +702,13 @@ func (ns *NodeServer) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandV
 	if volumeID == "" {
 		return nil, status.Error(codes.InvalidArgument, "volume ID must be provided")
 	}
-	volumePath := req.GetVolumePath()
+
+	// Get volume path
+	volumePath := req.GetStagingTargetPath()
+	if volumePath == "" {
+		// If Kubernetes version < v19.0
```

Comment:
it's better to add more details here what is changed in v1.19 and why this check is required

Reply:
I have added a comment in the PR, can I give the PR reference in the commit because otherwise it is difficult to explain that in the comments here.

Reply:
It would be better to explain the different here. Having it in the PR is good, but everyone reading this comment would then need to open the PR, and that seems unnecessary.

Reply:
fixed